Makefile

```diff
@@ -159,6 +159,7 @@ bundle: manifests kustomize
 	operator-sdk generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle --overwrite=false -q --version $(WMCO_VERSION) $(BUNDLE_METADATA_OPTS)
+	sed -i 's/REPLACE_IMAGE:latest/REPLACE_IMAGE/g' bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
```

Comment:
What does have to do with generated file linting?

Reply:
Without this change, in the verify script make bundle will be adding `:latest` to REPLACE_IMAGE, causing a diff to happen.

We do not want the `:latest` tag to be merged in, but our bundle generation script doesn't reflect that. So the options here were to either
1) change the bundle generation script like I did
2) change the validation script to remove the `:latest` when validating

If I went with 2 it would leave us with the extra work of manually removing the `:latest` tag, so 1 was the clear answer for me.

If you feel this needs to be broken into a separate PR I can do that.

Reply:
This is the sort of explanation that should be in the commit message 😃

>This is being manually removed when devs are merging bundle changes.

Please remind me why we manually remove this.

Reply:
> Please remind me why we manually remove this.
We need to remove it to maintain CI compatibility

Reply:
What do you mean by CI compatibility?

Reply:
CI doesn't recognize `:latest` after `REPLACE_IMAGE` and causes tests to fail with "error waiting for CSV to install"

Reply:
That's not CI but the SDK given we are using the SDK to install WMCO. Correct?